stdlib/ext/math-ext.mc

```diff
@@ -44,3 +44,6 @@ using eqf
 external externalAtan2 : Float -> Float -> Float
 let atan2 = lam x. lam y. externalAtan2 x y
 utest atan2 0. 1. with 0. using eqf
+
+external log : Float -> Float
```

Comment:
The log function implementation is part of my PR as well. I do think (which I guess is what you want to discuss) we should prefix externals with external and then wrap them in lambdas to allow currying, i.e. `let log = lam x. externalLog x`

Reply:
Oh, I see. For some reason I only saw the `external` definitions above, and not the `let`s. That sounds like good approach. I'll remove the addition here to avoid conflicts with your PR